x-pack/plugins/secops/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -54,23 +58,29 @@ describe('Host Summary Component', () => {
       );
       const wrapper = mountWithIntl(
         <ReduxStoreProvider store={store}>
-          <DragDropContextWrapper>{draggable}</DragDropContextWrapper>
+          <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+            <DragDropContextWrapper>{draggable}</DragDropContextWrapper>
+          </ThemeProvider>
         </ReduxStoreProvider>
       );
       expect(wrapper.text()).toBe('debian');
     });
 
     test('if it returns a FormattedRelative value', () => {
       const draggable = createDraggable(
-        '2019-01-28T22:14:16.039Z',
+        moment()
+          .subtract(1, 'days')
+          .format(),
```

Comment:
I would avoid basing a test on date time now such as a subtraction. Is it possible to use an absolute date time again?

Reply:
This test is for verifying that specified timestamp(s) are wrapped in a `FormattedRelative`. If using an absolute date you'd have to test against the `.text()` returned from a similarly configured `FormattedRelative`, or alternatively verify the wrapper itself.

Reply:
Can't you do a `moment(some_num).subtract(1, 'days').format()` though?

Reply:
Yeah, that'd work too. I've got a smaller PR ready to go -- I'll pull these in and do it that way.

Reply:
I might be a little bit too late to the party, but if you put a number, your relative date will change every day too. Or do I miss something here?

Reply:
No, that is correct and I am incorrect. Garrett in his other PR, showed me the 🌤 light: https://github.com/elastic/kibana/pull/29668/files#r252478633